satpy/etc/readers/insat3d_img_l1b_h5.yaml

```diff
@@ -0,0 +1,162 @@
+reader:
+  name: insat3d_img_l1b_h5
+  short_name: Insat 3d IMG L1B HDF5
+  long_name: Insat 3d IMG L1B HDF5
+  description: >
+    Reader for Insat 3d IMG level 1B data in Hdf5 format.
+    For documentation see: https://mosdac.gov.in/insat-3d-references .
+  status: Beta, navigation still off
+  supports_fsspec: false
+  sensors: [insat3d_img]
+  default_channels: [VIS, WV, TIR1, TIR2, SWIR, MIR]
+  reader: !!python/name:satpy.readers.yaml_reader.GEOFlippableFileYAMLReader
+
+file_types:
+    h5_std:
+        file_reader: !!python/name:satpy.readers.insat3d_img_l1b_h5.Insat3DIMGL1BH5FileHandler
+        file_patterns: [
+          "{platform:2s}{sensor:3s}_{nominal_time:%d%b%Y_%H%M}_{level:3s}_STD_V01R00.h5"
+        ]
+
+
+datasets:
+  VIS:
+    name: VIS
+    resolution: 1000
+    wavelength: [0.525, 0.65, 0.775]
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      radiance:
+        standard_name: toa_outgoing_radiance_per_unit_wavelength
+        units: mW.cm-2.sr-1.micron-1
```

Comment:
Not sure we standardized on it, but are spaces and periods both accepted as udunits/CF compliant units?

Reply:
good question! I'll look into it.

Reply:
using the `cfunits.Units` class seems to accept this:
```python
In [15]: 1000 * Units("mW.cm-2.sr-1.micron-1")
Out[15]: <Units: 10000000000 m-1.kg.s-3.rad-2>
```